### PR TITLE
ACM-992: Show errors on InfraEnv and Cluster details page

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

**Comment by mareklibra** [comment]
WIP screenshots:

---

---

---

File: src/cim/components/common/ResourceAlerts.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  }
+
+  return (
+    <Alert title={title} variant={AlertVariant.danger} isInline className="cim-resource-alerts">
+      <TextContent>
+        <TextList component={TextListVariants.ul}>
+          {conditions.map((c) => {
+            const [title, ...messages] = c.message.split(/:|\.,/);
+            return (
+              <>
```
Comment (rawagner) [inline src/cim/components/common/ResourceAlerts.tsx:30]: needs to have some kind of `key`

File: src/ocm/components/hosts/ClusterHostsTable.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ const ClusterHostsTable: React.FC<ClusterHostsTableProps> = ({
     () => [
       hostnameColumn(onEditHost),
       roleColumn(actionChecks.canEditRole, onEditRole),
-      statusColumn(AdditionalNTPSourcesDialogToggle, onEditHost),
+      statusColumn(AdditionalNTPSourcesDialogToggle, onEditHost), // Upravit tady pro ClusterDetail
```
Comment (rawagner) [inline src/ocm/components/hosts/ClusterHostsTable.tsx:41]: :)

File: src/cim/components/common/ResourceAlerts.tsx (hunk 1 above)
Comment (mareklibra) [reply src/cim/components/common/ResourceAlerts.tsx:30]: added

**Review by rawagner — DISMISSED** [review]

**Comment by mareklibra** [comment]
Rebased

**Review by rawagner — APPROVED** [review]

**Pull request merged**